graveler/graveler.go

```diff
@@ -926,19 +930,41 @@ func newStagingToken(repositoryID RepositoryID, branchID BranchID) StagingToken
 	return StagingToken(v)
 }
 
+func (g *Graveler) validateCommitParent(ctx context.Context, repositoryID RepositoryID, commit Commit) (CommitID, error) {
+	if len(commit.Parents) > 1 {
+		return "", ErrMultipleParents
```

Comment:
Can we just validate all parents? it will enable more options in the future, for add commit to head.
The caller can still verify min len of parents if there is a constraint.

Reply:
Not needed for now.. It will be easy to add in the future.